crates/utils/re_video/src/decode/ffmpeg_h264/ffmpeg.rs

```diff
@@ -516,68 +518,66 @@ fn write_ffmpeg_input(
 struct FrameBuffer {
     /// Received frame-infos, waiting to be matched to output frames.
     ///
-    /// Sorted by their presentation timestamp.
-    pending: BTreeMap<Time, FFmpegFrameInfo>,
+    /// Sorted by their frame number.
+    pending: BTreeMap<u32, FFmpegFrameInfo>,
 
-    /// Hightess DTC (Decoder timestamp) encountered so far.
-    highest_dts: Time,
+    /// The frame number of the next frame if we had any so far.
+    next_frame_nr: Option<u32>,
```

Comment:
Why is this an option? Is `Some(0)` even a valid state?

Wouldn't a normal `u32` starting at zero make more sense?

Reply:
Ah right, we may not be playing from the start. Might be worth a comment.